pkg/core/hostid/aws.go

```diff
@@ -25,6 +25,10 @@ func AWSUniqueID(cloudMetadataTimeout timeutil.Duration) string {
 		return ""
 	}
 	defer resp.Body.Close()
+	if resp.StatusCode == http.StatusNotFound {
+		log.Info("HTTP status 404, assuming not on EC2")
```

Comment:
Debug level. This will cause confusion at info level.

Reply:
I don't see it much different from the previous message.
If we move to debug then we potentially undid the previous fix of exposing errors without debug

Reply:
I'm not really concerned with a real AWS metadata service returning a 404, that would be a very strange scenario.

Reply:
Fair enough! it's just those corner cases that always come back to bite us.
I will push the change